### DISCUSSION TITLE
Add [GRIS] [Earth 3D] [KeyStroke Pro] [KeyCastr] [limbo] icons

### DISCUSSION THREAD

**Discussion by kuige** [elrumo/macOS_Big_Sur_icons_replacements#1120] [Icon request]
<img width="397" alt="image" src="https://user-images.githubusercontent.com/8212800/111890889-c7c78a00-8a28-11eb-9fd5-86bfe5d26874.png">
<img width="376" alt="image" src="https://user-images.githubusercontent.com/8212800/111890895-d31ab580-8a28-11eb-81fd-7d007492e75a.png">
<img width="194" alt="image" src="https://user-images.githubusercontent.com/8212800/111890901-dca41d80-8a28-11eb-9ac1-700c1044682e.png">

**Comment by cake03** [comment]
LIMBO
<img src="https://user-images.githubusercontent.com/80169337/125163764-544eb180-e1ac-11eb-93bc-1c94d7fa8b5c.png" height="512">
GRIS
<img src="https://user-images.githubusercontent.com/80169337/125164006-83b1ee00-e1ad-11eb-90dc-85edec5e11cf.png" height="512">

**Reply by cake03** [reply]
Keystroke Pro (you might not like it as much)
<img src="https://user-images.githubusercontent.com/80169337/125192732-f632d480-e266-11eb-8194-48a5bd925609.png" height="512" />

**Comment by QuintB** [comment]
I added two variants of KeyCastr!

Variant 1 is being approved
Variant 2 is [here](https://macosicons.com/u/Quint)

![126741475-fe00146b-f26a-41f5-be2f-7615316c0555](https://user-images.githubusercontent.com/50681031/126881216-94986f93-1e1b-4f7c-96cf-5a64de1c6868.png)
![126741647-a5e0ced2-8f5b-4695-9149-8ff3713d9a2d](https://user-images.githubusercontent.com/50681031/126881218-ed9c7b4b-a208-4de6-bf9f-405baf7c85f7.png)
